src/handlerUtil.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+/**
+ * src/handlerUtil.js
+ * Kafka handler utilities
+ */
+
+const logger = require('pino')();
+
+// The default handler just logs out the message
+const defaultHandler = (messageSet, topic, partition) => {
+  let messageSetOutput = '';
+  messageSet.forEach((m) => {
+    const key = m.message.key;
+    const value = m.message.value;
+    logger.info('Message from topic', topic, key, value);
```

Comment:
We should consider structuring this output a little better, e.g. both sumo and pino support log lines as json object, and having that kind of structure might be nice to have e.g. set up topic and key as "properties" on the pino log item... ?

Reply:
But we can make that a separate story if you want

Reply:
Yea I was thinking to make it a separate after we can spec out the log format in detail? Because I am not too sure how the later use case would work (storing logs in a big data store + pub/sub aggregation) and I guess the structure of the logs and how we handle them would depend on that?